app/views/questionnaire.py

```diff
@@ -530,6 +485,15 @@ def _build_template(current_location, context=None, template=None, display_navig
     return _render_template(context, current_location.block_id, front_end_navigation, metadata_context, current_location, previous_url, template)
 
 
+def _get_front_end_navigation(answer_store, completed_blocks, current_location, metadata):
+    if not current_location.is_interstitial():
```

Comment:
This is really confusing and needs changing, The function looks like

def is_interstitial(self):
        return self.block_id in ['introduction', 'summary', 'thank-you']

It would be very easy to call it thinking it covers all interstitial, however we use the term interstitial in an another manner to mean blocks inbetween other blocks with no questions on